### DISCUSSION TITLE
Waiting for streams, retrying every 1.0 seconds

### DISCUSSION THREAD

**Discussion by istenburg** [streamlink/streamlink-twitch-gui#965] [General]
Since today I can no longer watch streams via Streamlink Twitch Gui. It comes when starting any stream the following error:
`Waiting for streams, retrying every 1.0 seconds`
What's going on?
Did Twitch block this?

**Comment by bastimeyer** [comment]
https://github.com/streamlink/streamlink/issues/5370
